pyomo/checker/tests/examples/py3k/XRange_1.py

```diff
@@ -8,4 +8,7 @@
 #  This software is distributed under the 3-clause BSD License.
 #  ___________________________________________________________________________
 
+
+from six.moves import xrange
```

Comment:
I am not sure the intent of the file, but it also doesn't actually have any test invocations besides that it lives in a "test" folder. I'll remove the change, though.

Reply:
I will note, however, that this file originally lived somewhere else and was renamed in 2014. The two other files in that py3k directory simply print "Hello, World!", but wouldn't work in Python 3 (because there are no parentheses). So I can't help but wonder if they were originally just a "Py2 vs Py3" check? In which case... Are they even necessary?

Reply:
I really don't know.  The model checker was functionality that was created a long time ago by a student and to my knowledge has not been maintained.  I do not know if it even works any more.  @DLWoodruff or @whart222 may know / remember more.